nautobot/core/views/mixins.py

```diff
@@ -1105,3 +1082,26 @@ def notes(self, request, *args, **kwargs):
             "base_template": self.base_template,
         }
         return Response(data)
+
+
+def _get_bulk_pk_list(mixin, request) -> list:
```

Comment:
This could possibly be moved to be a method of `NautobotViewSetMixin`.

WDYT @glennmatthews

Reply:
I'd still prefer to revert this for now if at all possible - we have plans to change how bulk-update and bulk-delete handle the `pk_list` in both this file and in `nautobot.core.views.generic` in order to improve performance, and I don't have confidence yet that the same approach will necessarily be correct for both workflows.

Reply:
Reverted.